include/rdma/fi_ext.h

```diff
@@ -158,33 +158,37 @@ struct fid_peer_srx;
 struct fi_peer_rx_entry {
 	struct fi_peer_rx_entry *next;
 	struct fi_peer_rx_entry *prev;
+	struct fid_peer_srx *srx;
 	fi_addr_t addr;
 	size_t size;
 	uint64_t tag;
+	uint64_t flags;
 	void *context;
 	size_t count;
 	void **desc;
-	struct iovec iov[];
+	void *peer_context;
```

Comment:
maybe rename `peer_context` to `peer_resource` to be clearer?

Reply:
I think peer_context is clearer.  'context' is the term we use throughout the API.  How the peer uses that value is up to the peer.